app/controllers/metrics_controller.rb

```diff
@@ -51,8 +41,6 @@ def update
     end
   end
 
-  # DELETE /metrics/1
-  # DELETE /metrics/1.json
   def destroy
     @metric.destroy
     respond_to do |format|
```

Comment:
@dcox - Are any protections needed for the `destroy` and `update` actions? e.g. to stop one user deleting another user's metrics?